python3/Python3.g4

```diff
@@ -334,8 +334,12 @@ NEWLINE
    {
      String newLine = getText().replaceAll("[^\r\n\f]+", "");
      String spaces = getText().replaceAll("[\r\n\f]+", "");
+	 
+     // Strip newlines inside open clauses except if we are near EOF. We keep NEWLINEs near EOF to
+     // satisfy the final newline needed by the single_put rule used by the REPL.
      int next = _input.LA(1);
-     if (opened > 0 || next == '\r' || next == '\n' || next == '\f' || next == '#') {
+     int nextnext = _input.LA(2);
+     if (opened > 0 || (nextnext != -1 && (next == '\r' || next == '\n' || next == '\f' || next == '#')))
```

Comment:
Missing curly at the end? `{`

Reply:
Fixed, although it took me two shots to post it for some reason.